plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeSchemaSupport.java

```diff
@@ -126,6 +127,25 @@ public enum ColumnMappingMode
         /**/;
     }
 
+    public enum IsolationLevel
+    {
+        SNAPSHOTISOLATION("SnapshotIsolation"),
```

Comment:
From what I can tell in the docs snapshot isolation can never be used for writes. It is the behavior for reads only.